doc/smart-switch/pmon/smartswitch-pmon.md

```diff
@@ -86,9 +87,18 @@ The picture below highlights the PMON vertical and its association with other lo
 ### DPU startup sequence diagram
 <p align="center"><img src="./images/dpu-startup-seq.svg"></p>
 
-### Configuring startup and shutdown
-* The DPUs can be powered down by configuring the admin_status as shown.
-* The corresponding switch configDB table is also shown
+#### 2.1.1 DPUs in dark mode
+* Generally a smartswitch boots up with the DPUs in the dark mode with the help of platform config_db
+* NPU pmon listens to config DB events for DPU admin up/down events
+* PMON use set_admin_state() API to honor the request
```

Comment:
Does set_admin_state() power-on the DPUs from dark mode? Also, how can we query the current DPU state?

Reply:
set_admin_state() can power-on a DPUs from dark mode. "get_oper_status(self)" should return one of             "Offline", "Online",  "Partial Online"